src/index.ts

```diff
@@ -173,7 +185,11 @@ export default class IncludeFragmentElement extends HTMLElement {
     }
   }
 
-  #fetchDataWithEvents(): Promise<string> {
+  async #getStringData(): Promise<string> {
+    return (await this.#getData()).toString()
+  }
+
+  #fetchDataWithEvents(): Promise<string | TrustedHTMLStringable> {
```

Comment:
Would it be alright to turn this into an `async` function (and rewrite the `then` calls to use `await` instead) for easier maintainability?

Reply:
Done in https://github.com/github/include-fragment-element/pull/82